beets/importer.py

```diff
@@ -1034,8 +1034,8 @@ def handlers(cls):
             cls._handlers = []
             from zipfile import is_zipfile, ZipFile
             cls._handlers.append((is_zipfile, ZipFile))
-            from tarfile import is_tarfile, TarFile
-            cls._handlers.append((is_tarfile, TarFile))
+            from tarfile import is_tarfile, open
+            cls._handlers.append((is_tarfile, open))
```

Comment:
To avoid a potentially confusing collision with the built-in `open` function in this scope:

```suggestion
            import tarfile
            cls._handlers.append((tarfile.is_tarfile, tarfile.open))
```

Reply:
Done, thanks.